src/Kmd.Logic.CitizenDocuments.Client/CitizenDocumentsClient.cs

```diff
@@ -89,6 +94,144 @@ public async Task<CitizenDocumentUploadResponse> UploadAttachmentWithHttpMessage
             }
         }
 
+        /// <summary>
+        /// Uploads the single citizen document.
+        /// </summary>
+        /// <param name="document">Original citizen document.</param>
+        /// <param name="citizenDocumentUploadRequestModel">citizenDocumentUploadRequestModel to update to db.</param>
+        /// <returns>The fileaccess page details or error if isn't valid.</returns>
+        /// <exception cref="ValidationException">Missing cpr number.</exception>
+        /// <exception cref="SerializationException">Unable to process the service response.</exception>
+        /// <exception cref="LogicTokenProviderException">Unable to issue an authorization token.</exception>
+        /// <exception cref="CitizenDocumentsException">Invalid Citizen document configuration details.</exception>
+        public async Task<CitizenDocumentUploadResponse> UploadFileAttachmentDirectlyToStorageAsync(Stream document, CitizenDocumentUploadRequestModel citizenDocumentUploadRequestModel)
```

Comment:
`DirectlyToStorage` is imlementation detail and shouldn't be revealed to consumer.

Reply:
Changed method name